src/Application/Command/ReplayEventsCommand.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Application\Command;
+
+use App\Domain\Event\ValueObject\EventFilter;
+
+class ReplayEventsCommand
```

Comment:
This class need some work, like:

Add aggregate Id to replay, version from and version to. Maybe this last one optional.